lib/src/builders/expression.dart

```diff
@@ -210,8 +240,13 @@ abstract class AbstractExpressionMixin implements ExpressionBuilder {
   }
 
   @override
+  ExpressionBuilder increment([bool prefix = false]) {
+    return new _IncrementExpression(this, prefix);
```

Comment:
[nit] how are you choosing between `=>` and `return`? Why is this one different from `decrement` above?